drizzlepac/haputils/graph_utils.py

```diff
@@ -0,0 +1,258 @@
+""" Definition of Class to handle HAP SVM Bokeh Graphics
+
+"""
+import logging
+import sys
+import os
+import traceback
+import shutil
+
+from stsci.tools import logutil
+from astropy.io import fits
+import numpy as np
+
+from bokeh.plotting import figure
+from bokeh.models import ColumnDataSource, Label, Circle
+from bokeh.models.tools import HoverTool
+
+MSG_DATEFMT = '%Y%j%H%M%S'
+SPLUNK_MSG_FORMAT = '%(asctime)s %(levelname)s src=%(name)s- %(message)s'
+log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout,
+                            format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
+
+# Data columns from the Pandas dataframe which are to be used as common display value
+# for all hover tooltips.
+HOVER_BASIC_TIPS = [('Inst/Det', '@{gen_info.inst_det}'),
+                    ('Proposal ID', '@{gen_info.proposal_id}'),
+                    ('ASN ID', '@{header.ASN_ID}'),
+                    ('Filter', '@{gen_info.filter}'),
+                    ('ImageName', '@{gen_info.imgname}')]
```

Comment:
I would actually swap 'ImageName' and 'Inst/Det' in this list.  When reviewing the plot, the 'Inst/Det' will already be understood through the definition of the glyph color, while the user will want to know what image it corresponds with to look for any supplemental plots that may be available for that image.

Reply:
I see this has been updated in the code.